src/index.js

```diff
@@ -1,6 +1,14 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 
+const bool = 'bool';
+const number = 'number';
+const string = 'string';
+const json = 'json';
+const event = 'event';
+
+const Types = { bool, number, string, json, event };
```

Comment:
I would just create everything in the object literal instead of the separate variables.
```js
const Types = {
  bool: 'bool',
  ....
}
```